### PR TITLE
Azure-Cosmos version management HttpConstants

### PR DESCRIPTION
**Pull request by kushagraThapar**
* Removed hard coded version from HttpConstants
* Added azure-cosmos.properties file
* Added unit tests to verify current version

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/AzureCosmosPropertiesTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package com.azure.cosmos;
```
Comment (mbhaskar) [inline sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/AzureCosmosPropertiesTest.java]: Please add license header

Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/AzureCosmosPropertiesTest.java]: Good catch.

Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/AzureCosmosPropertiesTest.java]: Done.

File: sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/AzureCosmosPropertiesTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        String pomFileVersion = getVersionFromPomFile();
+        assertThat(HttpConstants.Versions.SDK_VERSION).isEqualTo(pomFileVersion);
+    }
+
+    private String getVersionFromPomFile() {
+        String fileName = "pom.xml";
+        String versionStartTag = "<version>";
+        String versionEndTag = "</version>";
+        File file = new File(fileName);
+        try {
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/AzureCosmosPropertiesTest.java]: ideally you can use xml parser to get the path of the filed of interest
anyway, not a big deal for a test and may not worth the time.

Thank you.

Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/AzureCosmosPropertiesTest.java]: Yeah, didn't want to add extra dependency :) 
but yes, I agree. We could use that.

File: sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/AzureCosmosPropertiesTest.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        assertThat(properties).isNotEmpty();
+        assertThat(properties.get("version")).isNotNull();
+        assertThat(properties.get("name")).isNotNull();
+    }
+
+    @Test(groups = "unit")
+    public void verifyProjectVersion() {
+        assertThat(HttpConstants.Versions.SDK_VERSION).isNotNull();
+        String pomFileVersion = getVersionFromPomFile();
+        assertThat(HttpConstants.Versions.SDK_VERSION).isEqualTo(pomFileVersion);
```
Comment (sima-zhu) [inline sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/AzureCosmosPropertiesTest.java]: This is cool. We do additional regex testing to test over the version format.

**Pull request merged**